packages/bs-neo3/src/services/vote/DoraVSNeo3.ts

```diff
@@ -0,0 +1,95 @@
+import { BSNeo3Helper } from '../../helpers/BSNeo3Helper'
+import { Account, BSCommonConstants, Network, Token } from '@cityofzion/blockchain-service'
+import { BSNeo3NetworkId } from '../../constants/BSNeo3Constants'
+import axios, { AxiosInstance } from 'axios'
+import { RpcVSNeo3 } from './RpcVSNeo3'
+import {
+  GenerateSigningCallbackResponse,
+  GetCandidatesToVoteResponse,
+  GetVoteDetailsByAddressResponse,
+} from '../../interfaces'
+
+type AxiosCandidate = {
+  scripthash: string
+  name: string
+  description: string
+  location: string
+  website: string
+  email: string
+  github: string
+  telegram: string
+  twitter: string
+  logo: string
+  votes: number
+  pubkey: string
+}
+
+export type AxiosGetCandidatesToVoteResponse = AxiosCandidate[]
+
+export type AxiosGetVoteDetailsByAddressResponse = {
+  vote: string
+  candidate: string
+  candidatePubkey: string
+  balance: number
+}
+
+type Params<BSName extends string> = {
+  network: Network<BSNeo3NetworkId>
+  feeToken: Token
+  validateAddress(address: string): boolean
+  generateSigningCallback(account: Account<BSName>): Promise<GenerateSigningCallbackResponse>
+}
+
+export class DoraVSNeo3<BSName extends string> extends RpcVSNeo3<BSName> {
```

Comment:
Rename to DoraVoteServiceNeo3. I think these shortenings we made only make things more difficult.